readme.rst

```diff
@@ -54,38 +54,19 @@ Basic Usage
 ****************************
 FARM offers two modes for model training:
 
-**Option 1: Run experiment(s) from config**::
+**Option 1: Run experiment(s) from config**
 
-    from farm.experiment import run_experiment, load_experiments
-    experiments = load_experiments("experiments/ner/conll2003_de_config.json")
-    run_experiment(experiments[0])
+.. raw:: html
+
+    <img src="docs/code_snippet_experiment.png">
```

Comment:
please use images hosted somewhere, since pypi will not find files inside folders (also adding them in manifest.in didnt work for images).
So for this image use exactly this address please: https://github.com/deepset-ai/FARM/blob/master/docs/code_snippet_experiment.png?raw=true